docs/language/source/specification_language_description.rst

```diff
@@ -921,7 +921,7 @@ Boolean describing whether the this group can be linked.
 ``dtype``
 ^^^^^^^^^
 
-String describing the data type of the dataset. Same as for attributes. See :numref:`sec-dtype` for details.
+String describing the data type of the dataset. Same as for attributes. See :numref:`sec-dtype` for details. ``dtype`` may be omitted for abstract classes. Best practice is to define ``dtype`` for most concrete classes.
 
```

Comment:
Minor clarification for the last two sentences. 

"``dtype`` may be omitted or set to ``null`` for abstract classes, indicating that the dataset or attribute may take on any valid data type. Best practice is to define ``dtype`` for concrete classes."